README.rst

```diff
@@ -203,7 +203,7 @@ page or ask us at our active Gitter channel, where we have maintainers from
 all over the world. We appreciate any help!
 
 We also have a
-`newcomer guide <http://coala.readthedocs.io/en/latest/Developers/Newcomers_Guide.html>`__
+`newcomer guide <http://api.coala.io/en/latest/Developers/Newcomers_Guide.html>`__
 to help you get started by fixing an issue yourself! If you get stuck anywhere
```

Comment:
The link is not shown, but if Newcomers guide is moved to a different URL in the future (it has already once), we don't want to have to go through the documentation and change the URL.
But rather we only will need to change the redirection of coala.io/Newcomers to a different URL.